README.md

```diff
@@ -74,10 +91,8 @@ user.cacher.something_you_want_to_cache
 
 ## Examples
 
-### Example 1: Cache the number of active user
-
-After specifying the name as `active_count` and how to get data when cache miss by lambda `User.active.count`.
-You could access the cached data by calling `active_count` method on the cacher, `User.cacher`.
+### Example 1: Cache the number of active users
+Specify the variable name as `active_count`. After using lambda `User.active.count` to define how the data can be accessed when there is a cache miss, you can get the cached data by calling `active_count` method on the cacher `User.cacher`.
```

Comment:
`active_count` is not a variable name. It's more like a method name of `cacher`

This line (`cache_at :active_count, ->{ active.count }`) will define a method named `active_count` to cacher.
You could call it like `cacher.active_count`

Reply:
"Specify the method name as `active_count`" sounds better? 
Initially, it was "...specifying the name as `active_count`"
I just wanted to make sure it is clearly specified what name the sentence is referring to.

Reply:
I think so. It's better to specify what the name is.